app/views/layouts/application.html.erb

```diff
@@ -36,6 +36,7 @@
     )%>);
 
     new ELMO.Views.MissionChangeDropdown({el: '#change_mission select'});
+    new ELMO.Views.RegenerableFieldView();
```

Comment:
But I'd somehow have to avoid instantiating more than one per page, right?

I suppose we could move to passing the element of each individual "Regenerate" field to its own `RegenerableFieldView` instance.